### PR TITLE
Map 1818 template kotlin merge

### PR DESCRIPTION
**Pull request by marcusaleman**
Build docker fix

### CODE REVIEW

File: helm_deploy/hmpps-uof-data-api/Chart.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ name: hmpps-uof-data-api
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: "2.9"
```
Comment (marcusaleman) [inline helm_deploy/hmpps-uof-data-api/Chart.yaml:8]: Change aligned to template and location-api

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
```
Comment (marcusaleman) [inline gradle/wrapper/gradle-wrapper.properties:3]: Change aligned to template and location-api

File: build.gradle.kts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_21
```
Comment (marcusaleman) [inline build.gradle.kts:1]: Change aligned to location-api and to fix DOCKER_BUILD issue.

File: applicationinsights.dev.json [hunk 4]
```diff
@@ -14,22 +14,19 @@
     "destination": "console"
   },
   "sampling": {
-    "percentage": 100
```
Comment (marcusaleman) [inline applicationinsights.dev.json:17]: Change aligned to location-api and template

File: .circleci/config.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@8
```
Comment (marcusaleman) [inline .circleci/config.yml:4]: Change aligned to location-api and template kotlin

File: Dockerfile [hunk 6]
```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM eclipse-temurin:21-jre-jammy AS builder
```
Comment (marcusaleman) [inline Dockerfile:1]: Change aligned to location-api and to fix DOCKER_BUILD issue.

**Review by michalnawrockidj — APPROVED** [review]
LGTM

**Pull request merged**